hack/test-end-to-end.sh

```diff
@@ -127,48 +169,48 @@ echo "[INFO] Pre-pulling and pushing centos7"
 docker pull centos:centos7
 echo "[INFO] Pulled centos7"
 
-docker tag centos:centos7 ${DOCKER_REGISTRY_IP}:5001/cached/centos:centos7
+docker tag -f centos:centos7 ${DOCKER_REGISTRY_IP}:5001/cached/centos:centos7
 docker push ${DOCKER_REGISTRY_IP}:5001/cached/centos:centos7
 echo "[INFO] Pushed centos7"
 
 # Process template and apply
 echo "[INFO] Submitting application template json for processing..."
-osc process -n test -f ${FIXTURE_DIR}/application-template-${BUILD_TYPE}build.json > "${CONFIG_FILE}"
+osc $OSC_OPTS process -n test -f ${FIXTURE_DIR}/application-template-${BUILD_TYPE}build.json > "${CONFIG_FILE}"
 # substitute the default IP address with the address where we actually ended up
 # TODO: make this be unnecessary by fixing images
 # This is no longer needed because the docker registry explicitly requests the 172.30.17.3 ip address.
 #sed -i "s,172.30.17.3,${DOCKER_REGISTRY_IP},g" "${CONFIG_FILE}"
 
 echo "[INFO] Applying application config"
-osc apply -n test -f "${CONFIG_FILE}"
+osc $OSC_OPTS apply -n test -f "${CONFIG_FILE}"
 
 # Trigger build
 echo "[INFO] Invoking generic web hook to trigger new build using curl"
-curl -X POST http://localhost:8080/osapi/v1beta1/buildConfigHooks/ruby-sample-build/secret101/generic?namespace=test && sleep 3
+curl -k -X POST $API_SCHEME://$API_HOST:$API_PORT/osapi/v1beta1/buildConfigHooks/ruby-sample-build/secret101/generic?namespace=test && sleep 3
```

Comment:
is "-k" well known by everyone else?  I think --insecure is more readable.  Also, why not specify the CA?